CHANGELOG.txt

```diff
@@ -0,0 +1,10 @@
+Changelog
+=========
+
+0.0.16 (2017-07-07)
+-------------------
+- Initial release
+
+0.0.17 (2017-11-28)
+-------------------
+Correctly scale the flame graph so it shows all the samples.
```

Comment:
These are in the incorrect order. Newest versions always come first in the file, so 0.0.17 should be above 0.0.16.

Reply:
Items under a release heading should each start with a leading hyphen.